.travis.yml

```diff
@@ -26,15 +26,14 @@ jobs:
         - .travis/export-github-repo.sh explorer/ explorer
 
     - &release-artifacts
-      if: type IN (api, cron) OR tag IS present
+      if: type = pull_request AND branch = master
       name: "macOS release artifacts"
       os: osx
       language: rust
       rust:
         - stable
       install:
         - source ci/rust-version.sh
-        - brew install coreutils
         - PATH="$PATH:/usr/local/opt/coreutils/libexec/gnubin"
```

Comment:
This needs to be exported, and in the `script` section.  I fought similar for the Ledger app CI

Reply:
I think that 3f78e91a275127af0132e96f816d161c2ec24d7c is all that's needed actually